hnginternship5/ziki
# STORY ID [404 pages]

#### What does this PR do?
* It creates the 404 page
#### Description of Task to be completed?
*  page `404.html` is created
* Styling is added to `404-page.css`
#### How should this be manually tested?
* Open the site on your browser
* Type localhost/404
* You should see the 404 page
#### Any background context you want to provide?
* use gulp to automate builds
#### What are the relevant pivotal tracker stories?
[#165477920](https://www.pivotaltracker.com/story/show/123456)
#### Screenshots (if appropriate)
![Screen Shot 2019-04-20 at 11 14 28 AM](https://user-images.githubusercontent.com/38491485/56457320-bb37e400-6370-11e9-93f8-cba2691607a2.png)
Tab
![Screen Shot 2019-04-20 at 11 14 17 AM](https://user-images.githubusercontent.com/38491485/56457323-c12dc500-6370-11e9-8cc8-2e17dc53e555.png)
Phone
#### Questions/Comments:
